README

```diff
@@ -58,9 +58,9 @@ References -
 [1] http://www.lothar.com/tech/crypto/
 [2] http://www.aet.tu-cottbus.de/personen/jaenicke/postfix_tls/prngd.html
 [3] http://www.gzip.org/zlib/
-[4] http://www.openssl.org/
-[5] http://www.openpam.org
-    http://www.kernel.org/pub/linux/libs/pam/
+[4] https://www.openssl.org/
```

Comment:
Fixing this in the main openssh repo (https://github.com/openssh/openssh-portable/blob/master/README) helps broader audience. You can raise a bug https://bugzilla.mindrot.org.


fyi, we frequently pull the changes from the openssh repo.